aptos-move/block-executor/src/executor.rs

```diff
@@ -240,7 +240,12 @@ where
             },
             ExecutionStatus::Abort(err) => {
                 // Record the status indicating abort.
-                ExecutionStatus::Abort(BlockExecutionError::FatalVMError((err, idx_to_execute)))
+                ExecutionStatus::Abort(BlockExecutionError::FallbackToSequential(PanicOr::Or(
```

Comment:
why do we just not return exception here? is any of the work below useful, if we are going to fallback to sequential anyways?

Reply:
This might still be speculative

Reply:
but it shouldn't happen, so even if there was a speculative Abort, and got cleaned up, we would want to know about it. Especially now with the fallback that's absolutely the best way to handle it, and would also simplify commit flow. Great, I'll try that.